### DISCUSSION TITLE
Wrong Language in HA

### DISCUSSION THREAD

**Discussion by Milo-PRT** [Thomas55555/husqvarna_automower#418] [Q&A]
Hello all
I have noticed for some time that the display language does not correspond to the language set in HA. The texts and messages are displayed in English, even though HA is set to German. 
%A is set in configuration….

what could be the reason for this?
Thanks for a good idea.

**Comment by Thomas55555** [comment]
I'm not sure
I also have this problem with native ha components. E.g. the thermostat shows `heat` instead of `Heizen`
